doc/man/scons.xml

```diff
@@ -2156,8 +2177,57 @@ repositories are searched in the order specified.</para>
 
 <refsect1 id='configuration_file_reference'>
 <title>CONFIGURATION FILE REFERENCE</title>
-<!--  .SS Python Basics -->
-<!--  XXX Adding this in the future would be a help. -->
+
+<refsect2 id='configuration_files'>
+<title>Configuration Files</title>
```

Comment:
Configuration files is likely going to be confusing.  I'd propose SConstruct/SConcript files instead.

Reply:
I can do that, struggling with right term.  Since the containing section was called "Configuration File Reference" I just picked Configuration Files. Happy to change both.